src/Login/index.tsx

```diff
@@ -24,52 +24,55 @@ export const LoginScreen = (props: LoginProps) => {
 
   const handleLogin = async () => {
     setIsLoading(true);
-    const userDetails = await login(email, password);
-    if (userDetails) {
-      toast.show({
-        title: 'Logged In',
-        status: 'success',
-        description: 'Enjoy !',
-      });
-      setIsLoading(false);
-      return props.setUser({
-        id: userDetails.id,
-        username: userDetails.username,
-        email,
-        token: userDetails.access_token,
-        isConnected: true,
-      });
-    }
-    setIsLoading(false);
-    return setErrorMessage("Your email and your password don't match");
-  };
 
-  const login = async (email: string, password: string) => {
     try {
-      const response = await fetch(`${API_URL}/auth/login`, {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify({
-          username: email,
-          password,
-        }),
-      });
+      login(email, password)
+        .then((response) => {
+          if (response.status >= 200 && response.status < 300) {
+            return response.json();
+          } else {
+            return setErrorMessage(
+              'Something has gone wrong with the server, please retry later',
```

Comment:
What is the error code returned by the server when the login/password pair is wrong?